wordpress-mobile/WordPress-Android
# Comment details pending moderation redesign

See https://github.com/orgs/Automattic/projects/944/views/1?pane=issue&itemId=60397816
Design yWt5gg3nWORhu079Qfv3mS-fi-1135_4600

This PR mainly implemented the redesign of pending moderation, but also removed lot of logic related to unused UI component.

| Screenshot 1 | Screenshot 2 |
|--|--|
| ![Screenshot_20240528-161610](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/6458d154-65c2-4642-8b51-54292f30fe37) | ![Screenshot_20240528-161624](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/a10a60c8-0100-4f59-8021-6565cc59d05f) |

-----

## To Test:

1. Sign in JP app
2. Go to `Notifications` tab
3. Click on a notification with comment type
4. It should display the UI of pending status, currently the `Approve comment` button is not actionable
5. Click on `More options`
6. It should display a bottom sheet of moderation, currently the buttons on the bottom sheet are not actionable
7. Go to My site -> More -> Comments
8. Click on a comment
9. Repeat step 4 - 6.
10. Done, thank you!

-----

## Regression Notes

1. Potential unintended areas of impact

    - notification, comment

11. What I did to test those areas of impact (or what existing automated tests I relied on)

    - manual

12. What automated tests I added (or what prevented me from doing so)

    - n/a

-----

## PR Submission Checklist:

Skipped

-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
